day_6/exercises/burrito.js

```diff
@@ -16,11 +16,29 @@ class Burrito {
     this.base = base;
     this.toppings = toppings;
   }
+
+  addTopping(Topping) {
```

Comment:
Be sure to utilize regular casing here. Anything that is PascalCase will read as a class.